docs/api-types/tls-route.md

```diff
@@ -0,0 +1,58 @@
+# TLSRoute API Reference
+
+## Introduction
+
+With integration of the Gateway API, AWS Gateway API Controller supports `TLSRoute`.
+This allows you to define and manage end-to-end TLS encrypted traffic routing to your Kubernetes clusters.
+
+### Considerations
+
+- `TLSRoute` sectionName must refer to an `TLS` protocol listener with `mode: Passthrough` in the parentRefs `Gateway`.
+- `TLSRoute` only supports to have one rule.
+- `TLSRoute` doesn't support any rule matching condition.
+- The `hostnames` field with exactly one host name is required. This domain name is used as a vpc lattice's Service Name Indication (SNI) match to route the traffic to the correct backend service.
```

Comment:
Is this essential? "This domain name is used as a vpc lattice's Service Name Indication (SNI) match to route the traffic to the correct backend service."

We explain this later with "The customer must use this domain name..." which I think is more helpful

Reply:
Ok, make sense to remove, just hope to emphasize, customer cannot use lattice generated DNS name to send traffic to backend service, they must use custom domain name

Reply:
Makes sense. I think we need to lean on the public Lattice documentation as well. The docs we have here in the controller don't have to be comprehensive and should focus on the integration rather than the core functionality Lattice provides.